origin-growth/src/models/growth_event.js

```diff
@@ -10,10 +12,21 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.ENUM(enums.GrowthEventTypes),
       status: DataTypes.ENUM(enums.GrowthEventStatuses),
       ethAddress: DataTypes.STRING,
-      data: DataTypes.JSONB
+      data: DataTypes.JSONB,
+      createdAt: {
+        type: DataTypes.DATE,
+        defaultValue: Sequelize.fn('NOW')
```

Comment:
Are there any events that do not have `date` associated with them? If not perhaps we can remove the `defaultValue` in favour of hard failing when there is an insert attempt without a date

Reply:
Currently there should always be a date. Implemented your suggestion to make date required.